num_nodes: 5000
step_size: 0.3
goal_threshold: 1
bias_probability: 0.1
radius: 0.3